packages/binding-mbus/src/mbus-client-factory.ts

```diff
@@ -0,0 +1,13 @@
+import { ProtocolClientFactory, ProtocolClient } from '@node-wot/core';
+import MBusClient from './mbus-client';
+
+export default class MBusClientFactory implements ProtocolClientFactory {
+  public readonly scheme: string = 'mbus+tcp';
+
+  public getClient(): ProtocolClient {
+    console.log(`MBusClientFactory creating client for '${this.scheme}'`);
```

Comment:
We have a [log convention](https://github.com/eclipse/thingweb.node-wot#logging) here. Can you please change this to:
```js
console.info("[binding-mbus]",`MBusClientFactory creating client for '${this.scheme}'`
```